Tests/TuistGeneratorTests/Generator/ProjectFileElementsTests.swift

```diff
@@ -589,49 +589,18 @@ final class ProjectFileElementsTests: TuistUnitTestCase {
         XCTAssertEqual(sdkElement?.name, sdk.path.basename)
     }
 
-    func test_generateDependencies_localSwiftPackage() throws {
```

Comment:
did this cause issues? my bad! I added it while working with @fortmarek on this. From what I saw, Xcode is quite happy to have local Swift Packages in any folder / group structure you give it.

If the goal is to have them separated from the Sources, perhaps they could be a flat list / mimic the file system within a top level `Packages` group instead?

Reply:
Ah yes, I removed the code in ProjectFileElements due to it creating some odd artifacts when generating. 

I copied the same way Xcode does when you add a new package (And XcodeProj) to include them at the top level - but I can look at changing this